src/03-modality-agnostic-files.md

```diff
@@ -197,6 +201,15 @@ for them:
     -   for "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`,
         `AMBIDEXTROUS`, `Ambidextrous`
 
+-   `strain`: string value indicating the strain of the species
+
+-   `strain_rrid`: research resource identifier ([RRID](https://scicrunch.org/resources/Organisms/search))
+    of the strain of the species
+
+-   `diagnosis`: string value describing the diagnosis of the participant.
```

Comment:
i don't know if this has to be a string value. in many datasets on openneuro diagnosis/dx is present and can be an enumerated type. also, this is one place, where one can have multiple designations depending on the study. we should allow for some notion of that, or simply remove diagnosis from this file.

Reply:
The aim of this PR is to add new columns to describe animal properties. In that context, I agree that diagnosis may be out of scope.

For context, I added the columns following this discussion https://github.com/bids-standard/bids-specification/issues/779#issuecomment-822660725, https://github.com/bids-standard/bids-specification/issues/779#issuecomment-822662620 and https://github.com/bids-standard/bids-specification/issues/779#issuecomment-823303226 because we also introduced pathology in `samples.tsv`.

Reply:
Following your suggestion, I removed the diagnosis column in 7146144 as being out of scope for this PR (animal properties).